facebook-share-widget.js

```diff
@@ -0,0 +1,44 @@
+reearth.ui.show(
+  `<style>
+      body { 
+        margin: 0;
+      }
+  </style>
+<!-- Facebook -->
+<div id="fb-root"></div>
+	<div class="fb-share-button" data-href="https://reearth.io/" data-layout="button" data-size="large" id="facebook-share">
+		<a target="_blank" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Freearth.io%2F&amp;src=sdkpreparse" class="fb-xfbml-parse-ignore">
+		Share
+		</a>
+	</div>
+</div>
+
+<script async defer crossorigin="anonymous" src="https://connect.facebook.net/ja_JP/sdk.js#xfbml=1&version=v13.0" nonce="QhVE4Z3L"></script>
+
+<script>
+// recieve message
+   window.addEventListener("message", e => {
+    if (e.source !== parent) return;
```

Comment:
```suggestion
    if (e.source !== parent || !e.data || e.data.type !== "facebook") return;
```

fix Twitter widget also